detox/test/e2e/copilot/08.copilot.location.test.js

```diff
@@ -0,0 +1,78 @@
+const {describeForCopilotEnv} = require("../utils/custom-describes");
+describeForCopilotEnv('Location', () => {
+
+    beforeEach(async () => {
+      await copilot.perform(
+        'Restart the React Native state',
+        'Navigate to the Location screen'
+      );
+    });
+
+    const tapGetLocationBtn = async () => {
+        await copilot.perform(
+          'Verify there is an element with the text "Get location"',
+          'Tap the get location element',
+          );
+      };  
+      
+    const setLocation = async() => {
+        await copilot.perform(
+            'Set the device location to (-80.125, 66.5)'
+        );
+    }
+
+    const setLocation2 = async() => {
+        await copilot.perform(
+            'Set the device location to (66.5, -80.125)'
+        );
+    }
```

Comment:
you can make it one method, that accepts the location as parameter

Reply:
also, the test coordinates might be better as consts (these are [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)), an [anti-pattern](https://en.wikipedia.org/wiki/Anti-pattern) of using numbers directly in source code).
It's not always bad to do such but since there are too many coordinates here so it might be a bit confusing

Reply:
Call it something like `fakeCoordinates1` and `fakeCoordinates2`, so it will be more semantic while reading the tests

Reply:
consider putting these steps **inline** (in the test and not in external function), now it's just text that is wrapped with method that has a very similar name

Reply:
According to our conversation, the driver was deleted. Instead, i added the suitable command to the test directly.
DUUMY_COORDINATE1 and DUUMY_COORDINATE2 was added.